lib/chef_zero/endpoints/policy_group_policy_endpoint.rb

```diff
@@ -0,0 +1,84 @@
+require 'ffi_yajl'
+require 'chef_zero/rest_base'
+require 'chef_zero/chef_data/data_normalizer'
+
+module ChefZero
+  module Endpoints
+    # /organizations/ORG/policy_groups/GROUP/policies/NAME
+    #
+    # in the data store, this REST path actually stores the revision ID of ${policy_name} that's currently
+    # associated with ${policy_group}.
+    class PolicyGroupPolicyEndpoint < RestBase
+
+      # GET /organizations/ORG/policy_groups/GROUP/policies/NAME
+      def get(request)
+        policy_name = request.rest_path[5]
+
+        # fetch /organizations/{organization}/policies/{policy_name}/revisions/{revision_id}
+        revision_id = parse_json(get_data(request))
```

Comment:
Does this parse the GET body?  If so, I'm surprised we send a GET request with data in the body